swiftwinrt/text_writer.h

```diff
@@ -460,6 +421,56 @@ namespace swiftwinrt
     };
 
 
+    template <typename T>
+    struct write_scope_guard
+    {
+        using writer_type = indented_writer_base<T>;
+        write_scope_guard(writer_type& w, std::string module = "", bool start_on_new_line = true) noexcept : m_writer(w), m_start_on_new_line(start_on_new_line), m_swift_module(module)
+        {
+        }
+
+        write_scope_guard(write_scope_guard const&) = delete;
+        write_scope_guard(write_scope_guard&& rhs) : m_writer(rhs.m_writer), m_lines(std::move(rhs.m_lines)) {}
```

Comment:
```suggestion
        explicit write_scope_guard(write_scope_guard&& rhs) : m_writer(rhs.m_writer), m_lines(std::move(rhs.m_lines)) {}
```

Reply:
i don't actually need to make changes to this file so i'm just going to revert this

Reply:
sike, not sure why i didn't think i needed this :)

Reply:
ok so what does making this explicit mean? does it require me to call `std::move` when i return the value from a function?

Reply:
It prevents single argument constructors from accidentally being used for type conversion.  I'm not sure if that's possible here, but rather than try to sort it out, it's best to just mark single argument ctor's as `explicit`.

Reply:
this is a move constructor, not a single argument constructor. i'd prefer to not mark this explicit, because it's weird to have to `std::move` return values